aaa_base.spec

```diff
@@ -33,12 +33,15 @@ BuildRequires:  git-core
 %endif
 
 Name:           aaa_base
-Version:        84.87%{?git_version}
```

Comment:
Are you sure this is needed? Even though with the current define above it doesn't matter the optional part seems to be more correct in case source validator becomes smarter at some point.